extension/EdFi.Ods.Extensions.EdFiLMS/Artifacts/MsSql/Structure/Ods/1000-lms-user-canvas.sql

```diff
@@ -0,0 +1,22 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE OR ALTER PROCEDURE lms.harmonize_lmsuser_canvas AS
+BEGIN
+    SET NOCOUNT ON;
+
+    UPDATE
+        lmsuser
+    SET
+        lmsuser.EdFiStudentId = edfistudent.Id
+    FROM
+        lms.LMSUser lmsuser
+    INNER JOIN
+        edfi.Student edfistudent
+    ON
+        lmsuser.SISUserIdentifier = edfistudent.StudentUniqueId
```

Comment:
Should this be `lmsuser.EdFiStudentId`?

Reply:
lmsuser.EdFiStudentId is what is being populated here, based on a match between the LMS SISUserIdentifier and the ODS StudentUniqueId.

Reply:
Oh right, I started thinking ahead to the script for populating the `edfilms.Assignment` table 😆 .